src/SmartComponents/Inventory/applications/compliance/SystemRulesTable.js

```diff
@@ -1,181 +1,364 @@
+        const { sortBy, rows, currentRows, columns, page, itemsPerPage } = this.state;
         const { loading } = this.props;
-        return (
-            <React.Fragment>
-                <Grid gutter="sm">
-                    <GridItem span={ 10 }>
-                        <Input
-                            id="search"
-                            type="text"
-                            style={ { width: '200px' } }
-                        />{ ' ' }
-                        <SearchIcon style={ { paddingTop: '4px' } } />
-                    </GridItem>
-                    <GridItem span={ 2 }>
-                        { !loading && <ComplianceRemediationButton selectedRules={ this.selectedRules() } /> }
-                    </GridItem>
-
-                    <GridItem span={ 12 }>
-                        <Table
-                            variant="large"
-                            header={ [ 'Rule', 'Reference ID', 'Policy', 'Severity', 'Passed' ] }
-                            hasCheckbox
-                            expandable
-                            onItemSelect={ this.onItemSelect }
-                            rows={ loading ? [ ...Array(10) ].map(() => ({
-                                cells: [{
-                                    title: <RowLoader />,
-                                    colSpan: 5
-                                }]
-                            })) : currentRows.map((oneRow, key) => ({
-                                ...oneRow,
-                                ...oneRow.hasOwnProperty('isOpen') ?
-                                    { isOpen: this.state.openNodes.indexOf(key) !== -1 } :
-                                    {}
-                            })
-                            ) }
-                            onExpandClick={ this.onExpandClick }
-                            footer={ <Pagination
-                                numberOfItems={ (this.state.rows.length - 2) / 2 }
-                                onPerPageSelect={ this.setPerPage }
-                                page={ this.state.page }
-                                onSetPage={ this.setPage }
-                                itemsPerPage={ this.state.itemsPerPage }
-                            /> }
-                        />
-                    </GridItem>
-                </Grid>
-            </React.Fragment>
-        );
+        if (loading) {
+            return (
+                <Table
+                    cells={ columns }
+                    rows={ [ ...Array(10) ].map(() => ({
+                        cells: [{
+                            title: <RowLoader />,
+                            colSpan: 5
+                        }]
+                    })) }>
+                    <TableHeader />
+                    <TableBody />
+                </Table>
+            );
+        } else {
+            return (
+                <React.Fragment>
+                    <TableToolbar>
```

Comment:
I'm not using `results` here because it puts the `rows.length` on a different row - that's not what I want according to the UI mocks I was given

Reply:
@dLobatog I'm not sure what the consensus is on that, because some apps have their results on a new row (Cost management, Remediations, Advisor/Insights)